docs/make.jl

```diff
@@ -113,3 +113,22 @@ makedocs(modules=[DiffEqBase,DiffEqProblemLibrary,DiffEqBiological],
 deploydocs(
    repo = "github.com/JuliaDiffEq/DiffEqDocs.jl.git"
 )
+
+#Redirect old links
+cd("build/") do
```

Comment:
Since you are not running in the docs directory:
```suggestion
cd(@__DIR__) do
```